src/components/Comments.jsx

```diff
@@ -14,6 +15,30 @@ export default function Comments({id, comments, setComments}) {
     })
     }, [])
 
+    function deleteCommentAPI(id) {
+        api.deleteCommentById(id)
+            .then(setDeleteError(false))
+            .catch(err => {
+                setDeleteError(true)
+                alert('error deleting! please refresh and try again!')
```

Comment:
I would personally have this as an error message within the comment itself, rather than an alert that forces the user to refresh. If the error has come from your server, forcing them to refresh to receive the same page is a bit of a faff.

If you did go down this route instead, you'd have to consider how to readd the comment you removed in the handleDelete function below so that users can try again without reloading the page entirely